packages/dapp-svelte-wallet/api/src/types.js

```diff
@@ -95,3 +95,24 @@
  * @property {any} requestContext
  * @property {string} id
  */
+
+/**
+ * @template T
+ * @typedef {Object} PetnameManager
+ * @property {(petname: Petname, object: T) => void} rename
+ * @property {(petname: Petname) => T} get
+ * @property { () => Array<T>} getAll
+ * @property {(petname: Petname, object: T) => void} add
```

Comment:
Though I didn't need it for most of the unit tests, I would prefer that `add` provide the same signature as `rename`:

```suggestion
 * @property {(petname: Petname, object: T) => Promise<void>} add
```